x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -189,26 +217,28 @@ export const MisconfigurationFindingsDetailsTable = memo(
         ),
       },
       {
-        field: 'result',
-        render: (result: CspFindingResult) => <CspEvaluationBadge type={result?.evaluation} />,
```

Comment:
why remove `CspFindingResult`? consider that in the future we might also need to support `unknown` or other stuff, its better to control this type from a single source

Reply:
oh thats a good point